Hardcoding /usr/bin/gmake instead of $MAKE where -f --am-depfiles is used

--- automake-1.16.1/m4/depout.m4	2018-02-26 21:38:27.000000000 +0000
+++ automake-1.16.1/m4/depout.m4.new	2019-08-25 14:59:06.741043451 +0000
@@ -62,4 +62,4 @@ AC_DEFUN([_AM_OUTPUT_DEPENDENCY_COMMANDS
 AC_DEFUN([AM_OUTPUT_DEPENDENCY_COMMANDS],
 [AC_CONFIG_COMMANDS([depfiles],
      [test x"$AMDEP_TRUE" != x"" || _AM_OUTPUT_DEPENDENCY_COMMANDS],
-     [AMDEP_TRUE="$AMDEP_TRUE" MAKE="${MAKE-make}"])])
+     [AMDEP_TRUE="$AMDEP_TRUE" MAKE="${MAKE-gmake}"])])
